src/main/scala/is/hail/expr/ir/Compile.scala

```diff
@@ -55,10 +82,10 @@ object Compile {
   }
 
   def apply[
-    T0 : TypeInfo : ClassTag,
-    T1 : TypeInfo : ClassTag,
-    T2 : TypeInfo : ClassTag,
-    R: TypeInfo : ClassTag
+  T0: TypeInfo : ClassTag,
```

Comment:
this seems like vaguely weird formatting, but I guess that's the default intellij formatting for this situation.